src/.vuepress/config.js

```diff
@@ -11,6 +11,7 @@ const sidebar = {
         '/guide/installation',
         '/guide/introduction',
         '/guide/instance',
+        '/guide/data-methods',
```

Comment:
**nitpick**: Could we move data-methods after template syntax so it will go before computed and watchers section?

Reply:
Done.

I don't feel strongly either way about this. Neither page leans heavily on the other so they could go in either order.

I've made some small tweaks to `data-methods.md` and `template.md` that seemed appropriate given the new order.

Reply:
Thank you! 👍🏻 Looks great now